package.json

```diff
@@ -100,7 +100,7 @@
     "tough-cookie": "2.3.2"
   },
   "lint-staged": {
-    "*.{js,jsx}:-flow-typed/**/*": [
+    "src/**/*.{js,jsx}": [
```

Comment:
Yeah I was trying to get this to work with multiple directories, but the minimatch glob pattern (https://github.com/isaacs/minimatch) is surprisingly cumbersome and I couldn't get it to do multiple directories _and_ multiple file extensions. I've opened a ticket to address this directly https://meetup.atlassian.net/browse/WP-340